### PR TITLE
initial work on adding django-countries to replace aputils Country model

### PR DESCRIPTION
**Pull request by davidhtien**
replaced `aputils.models.Country` with package `django-countries`: https://github.com/SmileyChris/django-countries

### CODE REVIEW

**Comment by ricefield** [comment]
@radicalray what do you think? I kind of prefer this, as it's cleaner and probably more thorough than what we had in place prior (I think the impetus behind this was to make bootstrapping the project a lot easier—no need for country fixture). do you have any concerns? maybe from the performance side?

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@ venv
 *.sublime-project
 *.sublime-workspace
 ap/media/badges/**
+ENV
+.vagrant
```
Comment (radicalray) [inline .gitignore]: get rid of these 2, since we're not using vagrant anymore

**Comment by davidhtien** [comment]
Hey can we get this merged in?

File: .gitignore (hunk 1 above)
Comment (davidhtien) [inline .gitignore]: I don't think its a big deal some people still have vagrant folders on their machines locally doesn't hurt to keep things like this in right?

**Comment by ricefield** [comment]
🙌

**Pull request merged**